sys/net/link_layer/ng_nomac/ng_nomac.c

```diff
@@ -40,7 +40,7 @@
  */
 static void _event_cb(ng_netdev_event_t event, void *data)
```

Comment:
If you really want to change the `nomac` implementation. Please do it in an other PR. Otherwise just take this changes out please.